app/grid/components/SelectedContext.css

```
@@ -87,3 +92,14 @@
 .more-details {
   padding: 16px;
 }
+
+
+.instructions {
+  background: #efefef;
+  border-radius: 6px;
+  padding: 16px;
+  word-wrap: break-word;
+  white-space: pre-wrap;
+  font-size: 12px;
+  line-height: 22px;
+}
```

```suggestion


.instructions {
  background: #eeeeee;
  border-radius: 6px;
  padding: 16px;
  word-wrap: break-word;
  white-space: pre-wrap;
  font-size: 12px;
  line-height: 22px;
}
```

change this color plz

---

agreed

---

agreed!